controllers/imagejob/imagejob_controller.go

```diff
@@ -63,8 +63,13 @@ var (
 	successDelDelaySeconds = flag.Int64("job-cleanup-on-success-delay", 0, "Seconds to delay job deletion after successful runs. 0 means no delay")
 	errDelDelaySeconds     = flag.Int64("job-cleanup-on-error-delay", 86400, "Seconds to delay job deletion after errored runs. 0 means no delay")
 	successRatio           = flag.Float64("job-success-ratio", 1.0, "Ratio of successful/total runs to consider a job successful. 1.0 means all runs must succeed.")
+	skipNodesSelectors     = nodeSkipSelectors([]string{"kubernetes.io/os=windows", "eraser.sh/cleanup.skip"})
 )
 
+func init() {
```

Comment:
How come this is in a `func init()`?

Reply:
I suppose it should be in the `var` block, with the other flags, in `main.go`. I'll move it.

Reply:
Sorry, I misread this...
flag.Var is fine here... my eyes were just seeing it as us setting a flag value here.

Reply:
See above, log.V(1).Info is a way to get debug-only logging here, as it raises the verbosity level by one. Since there's no .Debug() method, I wasn't sure how to go about that here.